src/main/kotlin/com/openlattice/shuttle/hazelcast/serializers/ShuttleSharedStreamSerializers.kt

```diff
@@ -0,0 +1,3 @@
+package com.openlattice.shuttle.hazelcast.serializers
+
+class ShuttleSharedStreamSerializers() {}
```

Comment:
🤔
Shuttle depends on `conductor-client` and `rhizome`, each of which should have access to a large set of our existing `*StreamSerializers`

Reply:
But the `Integration` class is in shuttle, and cond-cli can't depend on shuttle so the stream serializer for `Integration` can't go there.

Reply:
I think `Integration` should probably be moved to `conductor-client` then

Reply:
ping @pckeyes